bin/trigger_rc_tests.py

```diff
@@ -0,0 +1,125 @@
+
+    payload = {"branch": REPO_BRANCH}
+
+    if parameters is not None:
+        payload["parameters"] = parameters
+
+    circle_ci_conn.request(
+        method="POST", url=api_endpoint, body=json.dumps(payload), headers=headers
+    )
+    resp = circle_ci_conn.getresponse().read().decode("utf-8")
+    circle_ci_conn.close()
+    return resp
+
+
+def get_job_state(circleci_token: str, pipeline_id: str):
+    circle_ci_conn = http.client.HTTPSConnection(CIRCLECI_URL)
+    api_endpoint = f"/api/v2/pipeline/{pipeline_id}/workflow"
+
+    headers = {"content-type": "application/json", "Circle-Token": circleci_token}
+
+    circle_ci_conn.request(method="GET", url=api_endpoint, headers=headers)
+    resp = circle_ci_conn.getresponse().read().decode("utf-8")
+    circle_ci_conn.close()
+    return resp
+
+
+def main(circleci_token: str, astro_path: str):
+    # Getting Astronomer Helm Chart - FileName
+    file_list = os.listdir(astro_path)
+
+    astro_version = None
+    for file_name in file_list:
+        x = re.search("astronomer-.*.tgz", file_name)
+        if x is not None and astro_version is None and "rc" in file_name:
+            print(f"INFO: Found file {file_name}")
+            astro_version = file_name
+
+    if astro_version is None:
+        print(
+            f"INFO: Skipping calling workflow as no valid version found for the RC test. Below files are found at path: {astro_path}."
+        )
+        print(json.dumps(file_list))
+        raise SystemExit(0)
+
+    # Trigger the workflow
+
+    astro_version = astro_version.removeprefix("astronomer-")
+    astro_version = astro_version.removesuffix(".tgz")
+
+    parameters = {"astro_version": astro_version, "test_upgrade_workflow_gen": True}
+
+    print("INFO: Printing parameters")
+    print(json.dumps(parameters, indent=1))
+
+    # Run Workflow
+    run_workflow_resp = run_workflow(
+        circleci_token=circleci_token, parameters=parameters
+    )
+
+    pipeline_id = json.loads(run_workflow_resp)["id"]
+    pipeline_number = json.loads(run_workflow_resp)["number"]
+
+    # Printing Info
+    print(
+        f"CircleCI JOB URL = https://app.circleci.com/pipelines/github/{GITHUB_ORG}/{REPO}/{pipeline_number}"
+    )
+
+    print("INFO: Waiting until pipeline starts running. It will wait for 5 min.")
+    pipeline_state = "pending"
+    counter = 0
+    while "pending" != pipeline_state:
+        time.sleep(60)
+        job_state_resp = get_job_state(
+            circleci_token=circleci_token, pipeline_id=pipeline_id
+        )
+        pipeline_state = json.loads(job_state_resp)["items"][0]["status"]
+        counter = counter + 1
+
+        if counter == 5:
+            break
+
+    if "success" != pipeline_state and "running" != pipeline_state:
+        raise SystemError(1)
+    else:
+        raise SystemExit(0)
+
+
+if __name__ == "__main__":
+    arg_parser = argparse.ArgumentParser(description="Optional app description")
+
+    # Required positional argument
```

Comment:
Remove unneeded comment. `required=True` is sufficient information.

```suggestion
    # Required positional argument
```

Reply:
Done!